CONTRIBUTING.rst

```diff
@@ -99,6 +100,29 @@ If you are new to GitHub, below you can find a list of useful tutorials:
 - http://rogerdudler.github.io/git-guide/index.html
 - https://www.atlassian.com/git/tutorials
 
+Install pre-commit hooks
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you cloned the repository and the development environment is installed (and active),
+install the git pre-commit hook by executing the following command in the repository's
+root::
+
+    pre-commit hook install
+
+The pre-commit hooks are scripts that are executed in every commit automatically to
+identify simple issues with the code. If an issue is identified (the pre-commit script
+exits with non-zero status), the hook aborts the commit and print the error.
+Currently, Pysteps only test the code to be committed complies with black's autoformat.
+In case that the commit is aborted, you only need to run black in the entire source code.
+This can be done by running `black .` or `pre-commit run --all-files`.
```

Comment:
I think the commands in this line are not rendering as expected in the documentation: 
"This can be done by running *black .* or *pre-commit run –all-files*. "
instead of
"This can be done by running `black .` or `pre-commit run --all-files`. "

Just prepending `:code:` might solve the issue.